neo-codap/src/app-data.tsx

```diff
@@ -0,0 +1,28 @@
+import { types } from 'mobx-state-tree';
+
+export const CaseTableSortModelData = types.model('CaseTableSortModelData', {
+  colId: types.string,
+  sort: types.string
+});
+export type ICaseTableSortModelData = typeof CaseTableSortModelData.Type;
+
+export const CaseTableData = types.model('CaseTableData',
+  {
+    sortModel: types.array(CaseTableSortModelData)
+  })
+  .actions(self => ({
+    // tslint:disable-next-line:no-any
+    setSortModel(sortModel: any) {
+      self.sortModel = sortModel;
+    }
+  }));
+export type ICaseTableData = typeof CaseTableData.Type;
```

Comment:
My inclination would be to put the CaseTableData definitions in case-table.tsx and then have app-data.tsx import them rather than have app-data.tsx define them and import them the other way. Then down the road, the graph can define its own portion of the AppData rather than having app-data.tsx mingle data definitions for multiple components.

Reply:
Good point.  I think I want to rename them anyway after I started on the graph syncing since you already have a IGraphData interface defined.  I'll merge my new graph code into this PR and split the files and rename the types/instances.